GVFS/GVFS.Common/ModifiedPathsDatabase.cs

```diff
@@ -52,6 +52,34 @@ public static bool TryLoadOrCreate(ITracer tracer, string dataDirectory, Physica
             return true;
         }
 
+        public void Compress(ITracer tracer)

```

Comment:
I think this isn't so much "Compress" as it is "RemoveDuplicateEntries", right?

Reply:
Maybe "RemoveCoveredEntries" or "RemoveFolderEntries" because these really aren't duplicates.
`dir/file.txt` is not a duplicate of `dir/` but will be removed because of it.

Reply:
"Duplicate" in the sense that it adds no new behavior. "Compress" in the sense that we're removing redundant string and making the file smaller. Yea this can go either way, but some comments explaining its purpose would probably be helpful in this case since it's not something that's easy to capture in just a name.